test/passes/dwarfdump_roundtrip_dwarfdump.bin.txt

```diff
@@ -141,7 +141,7 @@ Abbrev table for offset: 0x00000000
               DW_AT_high_pc [DW_FORM_data4]	(0x00000000)
 
 0x00000026:   DW_TAG_subprogram [2]  
-                DW_AT_low_pc [DW_FORM_addr]	(0x0000000000000000)
+                DW_AT_low_pc [DW_FORM_addr]	(0x00000000ffffffff)
```

Comment:
This looks odd, but is what LLVM does, and LLVM's current dwarfdump (newer than the dumping code we are using here) will actually print `(dead code)` for this.